### ISSUE TITLE
a11y improvements for HeroImage

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1121]
HeroImage accessibility can improve in a few places: 

## HeroImage

- H3 tag should be h2  
- Graphics missing alts/aria-hidden
